cms/djangoapps/import_from_modulestore/models.py

```diff
@@ -0,0 +1,144 @@
+"""
+Models for the course to library import app.
+"""
+
+import uuid as uuid_tools
+from typing import Optional
+
+from django.contrib.auth import get_user_model
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from model_utils.models import TimeStampedModel
+from opaque_keys.edx.keys import UsageKey
+from opaque_keys.edx.django.models import (
+    LearningContextKeyField,
+    UsageKeyField,
+)
+from openedx_learning.api.authoring_models import LearningPackage, PublishableEntity
+
+from .data import ImportStatus
+
+User = get_user_model()
+
+
+class Import(TimeStampedModel):
+    """
+    Represents the action of a user importing a modulestore-based course or legacy
+    library into a learning-core based learning package (today, that is always a content library).
+    """
+
+    uuid = models.UUIDField(default=uuid_tools.uuid4, editable=False, unique=True)
+    status = models.CharField(max_length=100, choices=ImportStatus.choices, default=ImportStatus.PENDING, db_index=True)
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
+
+    source_key = LearningContextKeyField(help_text=_('The modulestore course'), max_length=255, db_index=True)
+    target = models.ForeignKey(LearningPackage, models.SET_NULL, null=True)
+
+    class Meta:
+        verbose_name = _('Import from modulestore')
+        verbose_name_plural = _('Imports from modulestore')
+
+    def __str__(self):
+        return f'{self.source_key} - {self.target}'
```

Comment:
```suggestion
        return f'{self.source_key} -> {self.target}'
```
suggestion for readability

Reply:
Added.